internal/event/target/kafka.go

```diff
@@ -73,6 +74,9 @@ const (
 	EnvKafkaClientTLSCert = "MINIO_NOTIFY_KAFKA_CLIENT_TLS_CERT"
 	EnvKafkaClientTLSKey  = "MINIO_NOTIFY_KAFKA_CLIENT_TLS_KEY"
 	EnvKafkaVersion       = "MINIO_NOTIFY_KAFKA_VERSION"
+	EnvKafkaBatchSize     = "MINIO_NOTIFY_KAFKA_BATCH_SIZE"
+
+	maxBatchLimit = 100
```

Comment:
Curious as to the reasoning for choosing 100 as the `maxBatchLimit`. I can potentially see wanting to go higher than this, although in a production setting where Kafka is deployed very close to MinIO 100 should be sufficient.

Reply:
the idea is to maintain a threshold on the async threads. we need to decide an optimal value. 100 seems to a "good-enough" value.